src/applications/find-forms/api/index.js

```diff
@@ -44,5 +49,8 @@ export const fetchFormsApi = async (query, options = {}) => {
     });
   }
 
-  return sortBy(onlyValidForms, 'id');
+  return {
+    allFormsTombstone,
```

Comment:
Instead of "allFormsTombstone" can we use something like "hasOnlyRetiredForms"? I like starting off with "has" because it suggests a yes/no value, and I like "retired" because I think it helps describe that a form has basically served its purpose and can't be submitted any longer (I sound a little nerdy with that explanation.)

Reply:
That's so nerdy, I love it.